tests/conftest.py

```diff
@@ -70,6 +71,23 @@ def admin_client(admin_user):
 
 
 @pytest.fixture()
+def staff_user(db):
+    """A Django staff user"""
+    return User.objects.create_user(
+        email='staff_test@example.com', password='password', is_staff=True,
+        is_active=True)
+
+
+@pytest.fixture()
+def staff_client(staff_user):
+    """A Django test client logged in as an staff member"""
+    from django.test.client import Client
+    client = Client()
```

Comment:
Instead of creating the client here you could use a fixture just as in `authorized_client` below.